### ISSUE TITLE
need consistent approach to publishing documentation

### ISSUE THREAD

**Issue by christo4ferris** [hyperledger/hyperledger#28]
We really need a consistent (and hyperledger branded) means of publishing documentation for the various projects. Right now, we are all over the map. STL publishes [http://intelledger.github.io/](http://intelledger.github.io/) and fabric has [markdown pages](https://github.com/hyperledger/fabric/tree/master/docs). I know that some of the IBMers wanted very much to emulate the approach taken by STL (it is very nice;-) There's also interest in hosting Swagger docs of the fabric apis. 

Would be nice to have a main page for the hyperledger project eg hyperledger.github.io from which we could link to incubating and formal projects, documentation etc.

**Comment by joshhus** [comment]
joshhus [10:53 AM] 
Looking at this with community member @jonathanlevi ... the desired output is Python / Sphinx / .rst source -- whereas GitHub is straight .md + HTML. We have gotten our .mds to .rst using pandoc (notice would require continuous builds). ... As a backup plan, GitHub's "Beautiful Docs" is the GitHub output. We can always use volunteers who want to be involved ...

**Comment by JonathanLevi** [comment]
Yes, I like the approach of having GitHub's simple markdown (.md) but also the combination of dynamically generated, well formatted 'static' HTMLs with CSS styling, etc.

We (with @joshhus) played last week with converting the current ".md" files using Pandoc and processed them with Sphinx. While I am doing these 'after hours' and can upload a 'cut' that is generated from the existing files we have.

Let me also put a few lines in #[1262](https://github.com/hyperledger/fabric/issues/1262)

**Comment by amecila** [comment]
@JonathanLevi can I lend a hand with files you haven't gotten to converting?

**Comment by christo4ferris** [comment]
@joshhus @JonathanLevi @amecila maybe you guys can discuss in slack or email and carve up the work
